jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -215,12 +215,23 @@
 			accessMode = lookupMode;
 		}
 		
+		/* For Java 15, the default is not to check for the "java.lang.invoke" package.
+		 * For earlier releases, these lookups are illegal
+		  */
+		private static boolean lookupJLIPackageCheckDefault(boolean doCheck) {
+			/*[IF Java15]
+			return false;
```

Comment:
Thanks @DanHeidinga , made following changes, please have another look.
1. Removed the helper constructor `Lookup(Class<?> lookupClass, int lookupMode, boolean doCheck)`;
2. Added comments to `Lookup(Class<?> lookupClass, boolean performSecurityCheck)` and stated that the incoming `performSecurityCheck` is expected to be false, otherwise `Lookup(Class<?> lookupClass)` can be used instead.
3. `lookupJLIPackageCheckDefault()` is used by three ctors to perform the package check for pre-JDK15 levels, and skip it for `JDK15` and beyond.